specs/0117-email-opt-in-unsubscribe.md

```diff
@@ -0,0 +1,97 @@
+
+## Standalone page for email consent shown after existing user log-in
+
+We need to make existing users aware of the new email communication preference.
+
+Upon login (after a user’s previous session expires) an email consent page should be displayed to existing users logging in for the first time after the consent feature is enabled. 
+
+They should be shown two buttons, one to opt-in to emails or one declining to opt-in. This should not be shown again if the user clicks either of the buttons, but should be shown again upon next login if the user closed their tab/browser.
+
+This page also should not be shown if the user has opted-in via their settings.
+
+(Section: content)
+- Header: "Update your email communication preferences"
+- Sub-header: "You can now choose your email communication preferences within Pontoon."
+- Body: "Would you like to receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform by email?”
+- Consent: “By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/en-US/privacy/websites/">Privacy Notice</a>."
+- Button: "Enable email updates”
+- Button: "No, thank you"
+- Body: “You can update your email communication preferences at any time in your settings."
+
+## New account creation email opt-in
+
+Ideally, we would like users to set up their email communication preferences right away upon successful account creation.
+
+After a user successfully creates their Pontoon account, display a standalone page which redirects to the homepage after the user enables or rejects email updates. This page informs the user of successful account creation and request for updating email communication preferences.
+
+They should be shown two buttons, one to opt-in to emails or one declining to opt-in. This should not be shown again if the user clicks either of the buttons, but the standalone page for email consent (above) should be shown again upon next login if the user closed their tab/browser.
+
+(Section: content)
+- Header: "Welcome to Pontoon!"
+- Sub-header: "You have successfully created your Pontoon account."
+- Body: "Would you like to receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform? This can be changed at any time in your settings."
+- Button: "Enable email updates”
+- Button: "No, thank you"
+- Consent: “By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/en-US/privacy/websites/">Privacy Notice</a>."
+
+## Additional card for tour
+
+As part of new user onboarding, point new users to user preferences and call out the ability to enable email communication preferences.
+
+(Section: content)
+- Header: Stay up to date
```

Comment:
```suggestion
- Header: "Stay up to date"
```

Maybe we should use backticks/code instead of quotes for text that represents actual UI text (buttons, field names, etc.)?